nginx-app/templates/default/scholar.conf.erb

```diff
@@ -1,5 +1,9 @@
 <% enable_caching = node.fetch('nginx-app', {}).fetch('browser_caching', {})['enabled'] %>
 <%= render_upstream(@php_upstream, @upstream_name) %>
+
+push_stream_shared_memory_size 32M;
```

Comment:
@fh Has to be in server: https://github.com/wandenberg/nginx-push-stream-module/blob/master/docs/directives/main.textile#push_stream_shared_memory_size